doc/classes/TreeItem.xml

```diff
@@ -118,7 +118,7 @@
 			<return type="TreeItem">
 			</return>
 			<description>
-				Returns the TreeItem's child items.
+				Returns the TreeItem's first child item or a null object if there is none.
```

Comment:
For consistency, we tend to say `null` rather than "a null object" in the class reference.

```suggestion
				Returns the TreeItem's first child item or [code]null[/code] if there is none.
```